app/views/patients/update.js.erb

```diff
@@ -0,0 +1,37 @@
+// rerack approx gestation
+$('#patient_appointment_date')
+  .parent()
+  .find('span.help-block')
+  .text('Approx gestation at appt: <%= @patient.last_menstrual_period_at_appt %>')
+
+// rerack status
```

Comment:
It'd be awesome to get that PR merged, I think it would make things even more snappy.